html_css/grid-lessons/grid-conclusion.md

```diff
@@ -14,7 +14,7 @@ One way to think about the different uses of Grid and Flexbox is to consider whe
 
 If, however, a developer is given a specific overall layout to adhere to and needs explicit placement of elements in two dimensions, using both rows **and** columns, then Grid would be much better suited. 
 
-Content-First and Layout-First Design doesn't mean that the developer should limit themselves to only using Flexbox or Grid individually! Let's look at something that can happen when we attempt to create a one dimensional grid - a task many reserve for Flexbox.
+Content-First and Layout-First Design doesn't mean that the developer should limit themselves to only using Flexbox or Grid individually! Let's look at something that can happen when we attempt to create a one dimensional grid - a task may reserve for Flexbox.
```

Comment:
Can you explain the reasoning behind this change?

"many" in the phrase "a task many reserve for flexbox" is pointing towards developers, I don't know if we can use the word "may" here.

Reply:
@01zulfi You are correct, this can be closed.